src/js/review/ArrayField.jsx

```diff
@@ -259,6 +256,11 @@ class ArrayField extends React.Component {
                 {_.get('ui:errorMessages.minItems', uiSchema) || 'You need to add at least one item.'}
               </div>
             </div>}
+          {title && !itemCountLocked &&
+            <div className="form-review-panel-page-header-row">
+              <button type="button" className="edit-btn add-btn primary-outline" onClick={() => this.handleAdd()}>Add Another</button>
```

Comment:
This represents some styling changes in Formation as well. 

@1Copenut please review this and LMK what you think. I've added the label update to another PR so this would say "Add Another Service Period" when that is done. 

![screen shot 2019-01-11 at 11 54 26](https://user-images.githubusercontent.com/4998130/51054113-f4ee9700-1598-11e9-81f1-68817b40f29e.png)